addons/website_sale/static/src/snippets/s_dynamic_snippet_products/000.scss

```diff
@@ -16,4 +56,19 @@
             --o-wsale-card-border-width: 1px;
         }
     }
+
+    // Applies arrows' alignment except when title is on the left + desktop
+    &:where(.o_wsale_products_opt_design_thumbs) {
+        &:not(:has(.s_dynamic_snippet_title_aside)) {
+            @include carouselControls;
+            @include colSizeRules;
+        }
+
+        @media (max-width: 992px) {
```

Comment:
Nope,`992px` is arbitrary. Works now and with the current setup,  but it will not when/if:
- The user customize the breakpoints
- Boostrap updates the breakpoints 

 Use the BS default breakpoints mixins instead.
https://getbootstrap.com/docs/5.3/layout/breakpoints/#media-queries